src/libs/Navigation/types.ts

```diff
@@ -483,19 +489,7 @@ type PublicScreensParamList = {
     [SCREENS.SAML_SIGN_IN]: undefined;
 };
 
-type AuthScreensParamList = {
-    [NAVIGATORS.BOTTOM_TAB_NAVIGATOR]: NavigatorScreenParams<BottomTabNavigatorParamList>;
-    [NAVIGATORS.CENTRAL_PANE_NAVIGATOR]: NavigatorScreenParams<CentralPaneNavigatorParamList>;
```

Comment:
Looks like part of the reason TS is failing is because you removed the central pane navigator by accident 😬

Reply:
confirmed, just adding this line back will fix your TS errors

Reply:
Thanks for helping me see that